holidays/countries/bhutan.py

```
@@ -142,26 +242,36 @@ def _populate_public_holidays(self):
+            self._add_holiday("Blessed Rainy Day", *dt)
 
         # Dassain.
         self._add_dussehra("Dassain")
 
         # Descending Day of Lord Buddha.
-        self._add_descending_day_of_lord_buddha("Descending Day of Lord Buddha")
+        dt = BhutanTibetanHolidays.DESCENDING_DAY_OF_LORD_BUDDHA_DATES.get(self._year)
+        if dt:
+            self._add_holiday("Descending Day of Lord Buddha", *dt)
 
     def _populate_subdiv_15_public_holidays(self):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_populate_subdiv_15_public_holidays`

Add return type annotation: `None`

(ANN202)